client/my-sites/woocommerce/dashboard/required-plugins-install-view.jsx

```diff
@@ -92,10 +93,14 @@ class RequiredPluginsInstallView extends Component {
 			toActivate: [],
 			toInstall: [],
 			workingOn: '',
-			progress: automatedTransferStatus ? transferStatusesToTimes[ automatedTransferStatus ] : 0,
+			progress:
+				automatedTransferStatus && transferStates.COMPLETE === automatedTransferStatus
```

Comment:
Does this change how the progress bar functions? I'm not really sure why this is necessary, but it does seem like a reasonable change.

Is this because this UI now only ever gets called when the site is already Atomic?

Reply:
Sorry, I forgot to comment on this change. Yes, it's related, and when the atomic state is `completed`, it essentially sets the progress to `41`, which will make the bar essentially > 100% at the very start. I noticed a logic error though, will be fixing that!

Reply:
Addressed logic error in a65ae7d